plugins/workbench-impl/package.json

```diff
@@ -12,7 +12,7 @@
     "test": "echo 'no test'",
     "lint": "eslint src",
     "lint:fix": "eslint --fix src",
-    "format": "prettier --write --plugin-search-dir=. 'src/**/*.{ts*,svelte}' && eslint --fix src",
+    "format": "prettier --write --plugin-search-dir=. src/**/*.{ts*,svelte} && eslint --fix src",
```

Comment:
' should not be removed. Since it will cause nested components to be non formatted.